modules/Bio/EnsEMBL/Analysis/Hive/Config/long_read.pm

```diff
@@ -0,0 +1,589 @@
+        dna_db         => $self->o('dna_db'),
+        indicate_index => $self->o('protein_blast_index'),
+        uniprot_index  => [ $self->o('protein_blast_db') ],
+        blast_program  => $self->o('uniprot_blast_exe_path'),
+        %{ get_analysis_settings( 'Bio::EnsEMBL::Analysis::Hive::Config::BlastStatic', 'BlastGenscanPep', { BLAST_PARAMS => { -type => $self->o('blast_type') } } ) },
+        commandline_params => $self->o('blast_type') eq 'wu' ? '-cpus=' . $self->o('use_threads') . ' -hitdist=40' : '-num_threads ' . $self->o('use_threads') . ' -window_size 40 -seg no',
+      },
+      -rc_name   => 'blast',
+      -flow_into => {
+        -1 => ['blast_long_read_10G'],
+        # 1 => ['intron_check'],
+      },
+    },
+
+    {
+      -logic_name => 'blast_long_read_10G',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveBlastRNASeqPep',
+      -parameters => {
+        input_db       => $self->o('long_read_collapse_db'),
+        output_db      => $self->o('long_read_final_db'),
+        dna_db         => $self->o('dna_db'),
+        source_db      => $self->o('long_read_collapse_db'),
+        target_db      => $self->o('long_read_final_db'),
+        indicate_index => $self->o('protein_blast_index'),
+        uniprot_index  => [ $self->o('protein_blast_db') ],
+        blast_program  => $self->o('uniprot_blast_exe_path'),
+        %{ get_analysis_settings( 'Bio::EnsEMBL::Analysis::Hive::Config::BlastStatic', 'BlastGenscanPep', { BLAST_PARAMS => { -type => $self->o('blast_type') } } ) },
+        commandline_params => $self->o('blast_type') eq 'wu' ? '-cpus=' . $self->o('use_threads') . ' -hitdist=40' : '-num_threads ' . $self->o('use_threads') . ' -window_size 40 -seg no',
+      },
+      -rc_name => 'blast10GB',
+      # -flow_into => {
+      #   1 => ['intron_check'],
+      # },
+    },
+
+    # TODO
+    # Added the following two analyses for completeness. Should they be deleted or edited and brought back to the pipeline?
+    # {
+    #   -logic_name => 'intron_check',
+    #   -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveHomologyRNASeqIntronsCheck',
+    #   -parameters => {
+    #     source_db => $self->o('long_read_blast_db'),
+    #     target_db => $self->o('long_read_final_db'),
+    #     intron_db => $self->o('rnaseq_refine_db'),
+    #     dna_db    => $self->o('dna_db'),
+    #   },
+    #   -rc_name   => '2GB',
+    #   -flow_into => {
+    #     1 => ['intron_check_10GB'],
+    #   },
+    # },
+
+    # {
+    #   -logic_name => 'intron_check_10GB',
+    #   -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveHomologyRNASeqIntronsCheck',
+    #   -parameters => {
+    #     source_db => $self->o('long_read_blast_db'),
+    #     target_db => $self->o('long_read_final_db'),
+    #     intron_db => $self->o('rnaseq_refine_db'),
+    #     dna_db    => $self->o('dna_db'),
+    #   },
+    #   -rc_name => '10GB',
+    # },
+
```

Comment:
This can be removed as we will not have a refine database at hand but also because the analysis is not needed